examples/protocols/multicolor-particle-calibration/multicolor-particle-calibration.py

```diff
@@ -409,10 +414,36 @@
                                              samples=read_wells4.output_pin('samples'),
                                              wavelength=sbol3.Measure(600, OM.nanometer))
 
-protocol.designate_output('measurements', 'http://bioprotocols.org/labop#SampleData', source=measure_fluorescence1.output_pin('measurements'))
-protocol.designate_output('measurements', 'http://bioprotocols.org/labop#SampleData', source=measure_fluorescence2.output_pin('measurements'))
-protocol.designate_output('measurements', 'http://bioprotocols.org/labop#SampleData', source=measure_fluorescence3.output_pin('measurements'))
-protocol.designate_output('measurements', 'http://bioprotocols.org/labop#SampleData', source=measure_absorbance.output_pin('measurements'))
+# FIXME Bryan derive this from the excel file via new primitive ExcelMetadata(file="measure_fluourescence1.xslx") and link its
+# output pin to meta1.metadata.descriptions
+# Change the format of the xarray to use dimensions, coordinates, and values as they appear in the xlsx.
+read_wells1_meta = json.dumps(xr.DataArray(
+    [[f"sample_{x}", measure_fluorescence1.name]
+     for x in get_aliquot_list(geometry="A1:B12")],
+    dims=("aliquot", "metadata"),
+    coords={"aliquot": get_aliquot_list(geometry="A1:B12"),
+            "metadata": ["sample_id", "measurement_type"]}).to_dict())
+
+meta1 = protocol.primitive_step("JoinMetadata",
+                              data=measure_fluorescence1.output_pin('measurements'),
```

Comment:
Shouldn't this have the metadata being added to it, and not just the data?
Same for the others below.